src/omero_zarr/raw_pixels.py

```diff
@@ -149,13 +154,25 @@ def plate_to_zarr(plate: omero.gateway._PlateWrapper, args: argparse.Namespace)
                 img = ws.getImage()
                 ac = ws.getPlateAcquisition()
                 ac_name = ac.getName() if ac else "0"
+                ac_names.add(ac_name)
+                paths.add(f"{ac_name}/{row}/{col}/{field_name}")
                 ac_group = root.require_group(ac_name)
                 row_group = ac_group.require_group(row)
                 col_group = row_group.require_group(col)
                 field_group = col_group.require_group(field_name)
                 n_levels = add_image(img, field_group, cache_dir=cache_dir)
                 add_group_metadata(field_group, img, n_levels)
             print_status(int(t0), int(time.time()), count, total)
+
+    plate_metadata = {
+        "rows": n_rows,
+        "columns": n_cols,
+        "row_names": list(row_names),
```

Comment:
I noticed that the row-names in the PR description are not ordered, which reminded me that `plate.listChildren()` isn't ordered, so the 'row_names' and 'col_names' won't be ordered.
I don't know that the client *need* these ordered but since they are a list it would be nice to order them.
Also I think it should be "column_names", not "col_names" since we have "columns" not abbreviated.